src/ngraph/frontend/onnx_import/op/conv_transpose.cpp

```diff
@@ -115,10 +138,14 @@ namespace ngraph
                     }
 
                     auto bias = inputs.at(2);
-
-                    return {std::make_shared<ngraph::opset0::Add>(
-                        conv_node,
-                        ngraph::op::make_broadcast_node(bias, conv_node->get_shape(), 1))};
+                    const Shape& new_shape = conv_node->get_shape();
+                    auto broadcasted_bias = std::make_shared<default_opset::Broadcast>(
```

Comment:
ONNX use biases with `[C]` shape so we need to broadcast it to the `output_shape` or reshape it to `[1,C,1,1]` (for 4D case) to perform an `add` operation since `numpy style` can't broadcast `[C]` into `[N, C, H, W]`.
ONNX doc:
```
B (optional) : T
    Optional 1D bias to be added to the convolution, has size of C.
```
IMHO we should adjust conv->bias fusion to support `bradcast bias` case as well as `[1, C, 1, 1]` since user can create a model with separate `convolution` and `add`(for bias) operations.

Reply:
I agree with @tsocha. NumPy broadcasting mechanism won't work for this case.

Reply:
Yes in ONNX it's just [C] and in converter we need to V1::Reshape it to [1,C,1,1] (for 4D case).
And then pass this reshaped constant to V1::Add(convolution, bias_reshape).

Reply:
Shouldn't we adjust the conv->bias fusion to support both cases? 
We can encounter user models which use broadcast bias case instead of [1, C, 1, 1]?

Reply:
In [N,C,H,W] cases we need to analyze tensor to detect that it's actually [1,C,1,1]. That is not hard to implement but we haven't seen such cases yet and I think won't see.
Also why I'm against [N,C,H,W] broadcasting is that if we have a sequence Conv->Add(NCHW)->BN first BN will be decomposed to linear ops and then Add(NCHW)->Mul(1,C,1,1)->Add(1,C,1,1)->Mul(1,C,1,1)->Add(1,C,1,1) sequence will be fused to the Mul(NCHW)->Add(NCHW) sequence that takes much more time in constant folding than just converting it to Mul(1,C,1,1)->Add(1,C,1,1). I can measure but we can avoid it with a simple solution: just using [1,C,1,1] for convolution biases.

Reply:
Another most critical reason is that Broadcast brakes network reshapability. For example if you want to change input shape for example from [N,C,H,W] to [N+1,C,H,W] where N > 1, this Broadcasted Constant with [N,C,H,W] shape won't be broadcasted to [N+1,C,H,W]. That's why you need to produce Add with [1,C,1,1] Constant shape.

Reply:
Just simply change Broadcast to V1::Reshape with [1,C,1,1] shape and you won't have this troubles.

Reply:
OK, changed.